### ISSUE TITLE
refresh and load not 

### ISSUE THREAD

**Issue by Osmancn** [peng8350/flutter_pulltorefresh#538]
refresh and load not trigger with mouse scroll on web(flutter 2.5.0)

**Comment by PcolBP** [comment]
Did you migrate using [this](https://flutter.dev/docs/release/breaking-changes/default-scroll-behavior-drag) guide after upgrading flutter?

**Comment by Osmancn** [comment]
> Did you migrate using [this](https://flutter.dev/docs/release/breaking-changes/default-scroll-behavior-drag) guide after upgrading flutter?

i did.scrolling is not problem but not trigger refresh and load action

**Comment by Osmancn** [comment]
https://user-images.githubusercontent.com/37872740/133758801-f32432d6-51b9-43d5-be8a-46fda957246b.mov

**Comment by stakan335** [comment]
hi, can you attach your code

**Comment by qingjixian** [comment]
flutter 2.5.0  https://flutter.dev/docs/release/breaking-changes/default-scroll-behavior-drag

Is there no adaptation?

mac os  loadmore  not work. skipCanRefresh = true,

Need to pull down manually！

On mac computers, three-finger pull-down can start and two-finger scrolling is not possible.

**Comment by laurentguenver-a** [comment]
I have the same problem. Any solution ? I must refresh to load more...

**Comment by vlladislav45** [comment]
@skyJinc i had similar problem to the web, the infinity scroll just didn't works so i just started to search about the issue and i saw this migration tutorial here: https://flutter.dev/docs/release/breaking-changes/default-scroll-behavior-drag . It helped to me to solve my problem but it would be better if that topic is added to the dependency's documentation.

**Comment by Shamik07** [comment]
---

View this: [flutter_pulltorefresh Pull Request #636](https://github.com/peng8350/flutter_pulltorefresh/pull/636)

**tldr;**
```dart
ScrollConfiguration(
  behavior: ScrollConfiguration.of(context).copyWith(
    dragDevices: {
      PointerDeviceKind.touch,
      PointerDeviceKind.mouse,
      PointerDeviceKind.trackpad,
      PointerDeviceKind.stylus,
    },
  ),
  child: SmartRefresher(
    enablePullDown: true,
    header: WaterDropHeader(),
    enablePullUp: true,
    footer: CustomFooter(
      builder: (BuildContext context, LoadStatus mode) {
        Widget body;
        if (mode == LoadStatus.idle) {
          body = Text("pull up load");
        } else if (mode == LoadStatus.loading) {
          body = CupertinoActivityIndicator();
        } else if (mode == LoadStatus.failed) {
          body = Text("Load Failed! Click retry!");
        } else if (mode == LoadStatus.canLoading) {
          body = Text("release to load more");
        } else {
          body = Text("No more Data");
        }
        return Container(
          height: 55.0,
          child: Center(child: body),
        );
      },
    ),
    controller: _refreshController,
    onRefresh: _onRefresh, // Check example above for this method
    onLoading: _onLoading, // Check example above for this method
    child: Offstage(), // Replace with your own widget
  ),
);
```

---
